src/components/ReviewDeposit.tsx

```diff
@@ -123,7 +123,7 @@ function ReviewDeposit({
             </span>
           </div>
         )}
-        {transactionData.txnGasCost?.valueUSD && (
+        {transactionData.txnGasCost?.valueUSD && shouldDisplayGas && (
```

Comment:
is `valueUSD` a BigNumber? If so, `shouldDisplayGas` is probably redundant and we can just do `valueUSD.eq(Zero)`

Reply:
good point. i did something similar using `gt` to avoid changing too much TS.